dask_match/expr.py

```diff
@@ -766,7 +766,9 @@ def _task(self, index: int):
         return (self.frame._name, self.partitions[index])
 
     def simplify(self):
-        if isinstance(self.frame, Blockwise):
+        if isinstance(self.frame, Blockwise) and not isinstance(
+            self.frame, BlockwiseIO
```

Comment:
This makes sense, because I *think* `BlockwiseIO` is the only case where we will end up getting the same expression back. Can also be more general and just verify that we are not just returning `self.frame` below.